### PR TITLE
Bah 4063 Database Specific Changes 

### PR DESCRIPTION
**Pull request by elder-joseph**
## Description
As a developer, I need to add a parent / child relationship so that the database is more normalized.

This PR is specific to the GIDS db source code, there is a separate PR associated with the app changes for 4063: #518 

https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/4063

## Testing done
Testing passes locally 

## Screenshots
N/A

## Acceptance criteria
- [x] Version table is created.
- [x] Has_many relationship is established from version to institution.
- [x] Has_many relationship is established from version to zipcode_rate.
- [x] Archiver is configured to archive institution and zipcode_rate data.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by zurbergram — APPROVED** [review]

**Review by sonntag-adam — APPROVED** [review]

File: db/schema.rb [hunk 1]
```diff
@@ -1615,8 +1617,13 @@
...
-    t.integer "version_id"
+    t.bigint "version_id"
     t.index ["version", "zip_code"], name: "zipcode_rates_archives_version_zip_code_idx"
+    t.index ["version_id"], name: "index_zipcode_rates_archives_on_version_id"
   end
 
+  add_foreign_key "institutions", "versions"
+  add_foreign_key "institutions_archives", "versions"
+  add_foreign_key "zipcode_rates", "versions"
+  add_foreign_key "zipcode_rates_archives", "versions"
```
Comment (jholton) [inline db/schema.rb]: Do these lines belong in `schema.rb`?

Comment (elder-joseph) [reply db/schema.rb]: Hi @jholton 

These lines get added to `schema.rb` upon executing the database migration script after making your recommended changes regarding indexing and foreign key constraints for this pull request.

Comment (jholton) [reply db/schema.rb]: You are correct!  My mistake.  I got confused because you can also use the `add_foreign_key` method in migrations.

**Review by jholton — APPROVED** [review]
LGTM

**Pull request merged**